src/main/java/opendcs/util/functional/ThrowingConsumer.java

```diff
@@ -1,10 +1,12 @@
 package opendcs.util.functional;
 
+import decodes.db.DatabaseException;
+
 /**
  * Function interval to allow for checked exceptions.
  */
 @FunctionalInterface
 public interface ThrowingConsumer<T,E extends Exception>
 {

```

Comment:
This is not correct here. This may be for other than just Database operations. Requiring everything to check for Database exception when there may be no database operations doesn't make sense.

While the situation certainly isn't perfect I've bee just catching the Non-SQL exceptions and rethrowing them with `new SQLException("some extra detail", ex)` where `ex` is that other exception.